lib/vquic/msh3.c

```diff
@@ -189,7 +211,8 @@ static void MSH3_CALL msh3_header_received(MSH3_REQUEST *Request,
               (int)Header->ValueLength, Header->Value);
   }
 
-  stream->recv_header_len += total_len - 1; /* don't include null-terminator */
+  stream->recv_header_len += total_len;
```

Comment:
Why this change?

Reply:
The lengths were off by one in my testing and the header were not correctly parsed.

Reply:
Ok. I guess we need some tests. 😄